domains/avishek.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "x0rzavi",
+        "email": "rock.nick.maverick@gmail.com"
+    },
+    "records": {
+        "CNAME": "x0rzavi.is-a.dev"
```

invalid CNAME
you might want to use URL for redirect to https://x0rzavi.is-a.dev
```suggestion
        "URL": "x0rzavi.is-a.dev"
```

---

I've updated it